docs/user/precheck/check_networks.rst

```diff
@@ -0,0 +1,342 @@
+IDs are valid.
+
+Possible errors
+^^^^^^^^^^^^^^^
+
+If check fails, :ref:`external networks mapping <external-networks-mapping>`
+will not work properly and such problems can be experienced:
+
+ - Incorrect Neutron :ref:`networking`;
+ - Errors during the :ref:`networking` that may result migration process
+   termination.
+
+How to resolve
+^^^^^^^^^^^^^^
+
+Simply review the :ref:`external networks mapping <external-networks-mapping>`
+configuration file and put there valid Neutron network IDs from source and
+destination clouds.
+
+
+External networks overlap
+-------------------------
+
+Overview
+^^^^^^^^
+
+Checks :ref:`external-networks-overlap`. Check starts only if
+``[migrate] migrate_ext_nets`` config option is set to ``True`` in the
+:ref:`primary-config-file`. This check verifies if there is any IP range
+overlapping between source and destination clouds' external networks. In
+particular, external subnets are compared. The main idea is to check if theirs
+allocation pools overlap. Only in this case migration will not start. If
+subnets have the same CIDRs, but different allocation pools, migration will
+proceed, because in this case there are no any IP conflicts.
+
+.. note::
+
+    This check will be skipped for all networks, defined in the
+    :ref:`external networks mapping <external-networks-mapping>` configuration
+    file, if :ref:`external networks mapping <external-networks-mapping>` is
+    enabled.
+
+Possible errors
+^^^^^^^^^^^^^^^
+
+Floating IPs are taken from the external networks’ pools. And with the
+overlapping ranges several absolutely the same Floating IPs can be allocated,
+that can lead to some routing problems.
+
+Example of overlap::
+
+    Network A (source cloud)
+    Subnet A
+    CIDR: 1.1.1.0/24
+    Allocation pool: 1.1.1.10 - 1.1.120
+
+    Network B (destination cloud)
+    Subnet B
+    CIDR: 1.1.1.0/25
+    Allocation pool: 1.1.1.5 - 1.1.1.25
+
+    Result:
+    Networks A and B overlap, cause they both have 1.1.1.10 - 1.1.1.25 IP range
+
+How to resolve
+^^^^^^^^^^^^^^
+
+Change allocation pool in any of these networks so that no overlapping IPs in
+it. Also :ref:`external networks mapping <external-networks-mapping>` can be
+used for such networks, if these networks should stay as is and match each
+other in this way.
+
+
+Networks segmentation IDs overlap
+---------------------------------
+
+Overview
+^^^^^^^^
+
+Checks if network's segmentation ID is available on the destination. If it is
+available (not busy with any other network), segmentation ID will be kept.
+Otherwise new segmentation ID will be generated for migrated network.
+
+.. warning::
+
+    This check always passes. If segmentation ID is already used on the
+    destination cloud, ``Warning`` message will be shown during the check and
+    migration will proceed.
+
+Possible errors
+^^^^^^^^^^^^^^^
+
+There are no any problems from the migration perspective, if such warnings are
```

Comment:
there are no any = In case such warning will be observed, this should not affect the migration.  Expected behavior in case Seg ID is used is already used in DST: migration tool will generate new unused segmentation ID. This can introduce a problem only in case users or applications uses hard coded Seg IDs in their configurations files, otherwise skip this warnings.